packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -16,7 +22,34 @@ This event type may be sent once per rate limiting period as a warning that some
 * `system.operation.rate_limit.notification`<br>
 This event type can provide additional information about rate limiting decisions. For example, this event might indicate that a violation event would have been emitted for a specific client rather than for a broader scope if you had chosen a different configuration.
 
-Additionally, there are specific [org-based System Log events](/docs/reference/api/system-log/#system-events) and [client-based System Log events](/docs/reference/rl-clientbased/#system-log-events). The org-based System Log events record system events related to your organization to provide an audit trail that you can use to understand platform activity and to diagnose problems. Client-based System Log events are fired when an individual client exceeds its assigned limit for the OAuth `/authorize` endpoint. Which event fired depends on the client-based rate-limiting mode that is set.
+### Org-based System Log event types
+
+The following org-based System Log events record system events related to your organization to provide an audit trail that you can use to understand platform activity and to diagnose problems.
+
+* `system.org.rate_limit.warning`<br>
+This event is sent when an endpoint is nearing its rate limit.
+
+* `system.org.rate_limit.violation`<br>
+This event is sent when an endpoint is exceeding its rate limit.
+
+* `core.concurrency.org.limit.violation`<br>
+This event is sent when a request is exceeding the org's allotted concurrent limit.
+
+### Client-based System Log event types
+
+The following client-based System Log events are fired when an individual client exceeds its assigned limit for the OAuth `/authorize` endpoint. Which event fired depends on the client-based rate-limiting mode that is set:
```

Comment:
The event that fires depends on the client-based rate-limiting mode that is set: